serverless/pages/custom-roles.mdx

```diff
@@ -0,0 +1,105 @@
+---
+slug: /serverless/custom-roles
+title: Custom roles
+description: Create and manage roles that grant privileges within your project.
+tags: [ 'serverless', 'Elasticsearch', 'Observability', 'Security' ]
+---
+
+{/* <DocIf condition={"((serverlessCustomRoles))" === "false"}>
+<DocBadgeComingSoon>Coming soon</DocBadgeComingSoon>
+</DocIf> */}
+
+{/* TO-DO: Change feature flag before merging */}
+{/* <DocIf condition={"((serverlessCustomRoles))" === "true"}> */}
+<DocBadge template="technical preview" />
+This content applies to: <DocBadge template="elasticsearch" slug="/serverless/elasticsearch/what-is-elasticsearch-serverless" /> <DocBadge template="observability" slug="/serverless/observability/what-is-observability-serverless" /> <DocBadge template="security" slug="/serverless/security/what-is-security-serverless" />
+
+The built-in <DocLink slug="/serverless/general/assign-user-roles" section="organization-level-roles">organization-level roles</DocLink> and <DocLink slug="/serverless/general/assign-user-roles" section="instance-access-roles">instance access roles</DocLink> are great for getting started with ((serverless-full)), and for system administrators who do not need more restrictive access.
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html#_roles */}
+
+{/*
+TBD: Accurate for serverless? Which types of users have access to this UI?
+The `manage_security` cluster privilege is required to manage custom roles.
+*/}
+As an administrator, however, you have the ability to create your own roles to describe exactly the kind of access your users should have.
```

Comment:
Just a suggestion. It is in the heading subtext, and implied by how to manage custom roles section, but more explicitly stating this might avoid some potential confusion.

```suggestion
As an administrator, however, you have the ability to create your own roles to describe exactly the kind of access your users should have within a specific project.
```